#! /bin/sh /usr/share/dpatch/dpatch-run
## 05_bevel_center.py_fix.dpatch by Antonio Ospite <ospite@studenti.unina.it>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: To fix a Syntax Error of the bevel_center.py script when using python 2.3

@DPATCH@
diff -pruN blender-2.41~/release/scripts/bevel_center.py blender-2.41/release/scripts/bevel_center.py
--- blender-2.41~/release/scripts/bevel_center.py	2006-01-16 22:42:53.000000000 +0100
+++ blender-2.41/release/scripts/bevel_center.py	2006-02-07 10:25:55.000000000 +0100
@@ -178,7 +178,7 @@ def make_faces():
 						NF.append(NMesh.Face([newV[ind2-2][0],newV[ind2-2][1],newV[ind2-1][0],newV[ind2-1][1]]))
 					
 					else:
-						ind2 = min(((newV[i][1].co-newV[i-1][0].co).length, i) for i in enumV)[1]
+						ind2 = min([((newV[i][1].co-newV[i-1][0].co).length, i) for i in enumV])[1]
 						NF.append(NMesh.Face([newV[ind2-1][1],newV[ind2][0],newV[ind2][1],newV[ind2-2][0]]))
 						NF.append(NMesh.Face([newV[ind2-2][0],newV[ind2-2][1],newV[ind2-1][0],newV[ind2-1][1]]))
 				
@@ -373,7 +373,7 @@ def bevel():
 	me = ob.getData()
 
 	NV = {}
-	NV_ext = set()
+	NV_ext = []
 	NE = {}
 	NC = {}
 	old_faces = []
